examples/api/lib/widgets/basic/indexed_stack.0.dart

```diff
@@ -0,0 +1,152 @@
+            ),
+            onSubmitted: (String value) {
+              setState(() {
+                names.add(value);
+              });
+              fieldText.clear();
+            },
+            controller: fieldText,
+          ),
+        ),
+        const SizedBox(height: 50),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == 0) {
+                  index = names.length - 1;
+                } else {
+                  index--;
+                }
+              }),
+              child: const Icon(key: Key('gesture1'), Icons.chevron_left),
+            ),
+          ],
+        ),
+        const SizedBox(height: 50),
+        Row(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == 0) {
+                  index = names.length - 1;
+                } else {
+                  index--;
+                }
+              }),
+              child: const Icon(key: Key('gesture1'), Icons.chevron_left),
+            ),
+            Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: <Widget>[
+                IndexedStack(
+                  index: index,
+                  children: <Widget>[for (String name in names) PersonTracker(name: name)],
+                )
+              ],
+            ),
+            GestureDetector(
+              onTap: () => setState(() {
+                if (index == names.length - 1) {
+                  index = 0;
+                } else {
+                  index++;
+                }
+              }),
+              child: const Icon(key: Key('gesture2'), Icons.chevron_right),
+            ),
+          ],
+        )
+      ],
+    );
+  }
+}
+
+class PersonTracker extends StatefulWidget {
+  const PersonTracker({super.key, required this.name});
+  final String name;
+  @override
+  State<PersonTracker> createState() => _PersonTrackerState();
+}
+
+class _PersonTrackerState extends State<PersonTracker> {
+  int counter = 0;
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      key: Key(widget.name),
+      decoration: BoxDecoration(
+          color: const Color.fromARGB(255, 239, 248, 255),
+          border: Border.all(color: const Color.fromARGB(255, 54, 60, 244)),
+          borderRadius: const BorderRadius.all(Radius.circular(10))),
+      padding: const EdgeInsets.all(16.0),
+      child: Column(
+        children: <Widget>[
+          Text('Name: ${widget.name}'),
+          Text('Score: ${counter.toString()}'),
+          TextButton.icon(
+            icon: const Icon(Icons.add),
+            onPressed: () {
+              setState(
+                () => counter++,
+              );
```

Comment:
I suppose this is "legal" according to the style guide, but don't format like this for `setState` calls. They're usually formatted like this:

```suggestion
              setState(() {
                counter++;
              });
```